docs/developers/architecture/app_model.md

```diff
@@ -4,64 +4,198 @@
 
 ```{important}
 **This is not a part of the public napari interface!**
-
-This page is mostly aimed at developers who are interested in contributing to
-or understanding the inner workings of napari.
 ```
 
 ```{warning}
 **Work in progress!**
-This document is here to give a little guidance on the current
-vision for how this application model might be used. It will very likely change
-as we develop it.
+The napari application model is currently being developed. This document is here to
+give guidance but things may change as we develop. It may link to current
+pull requests and issues which have more up to date information on specific
+areas.
 ```
 
 ## App-model
 
-The global application singleton can be retrieved with `napari._app_model.get_app()`.  It is an instance of
-[`app_model.Application`](https://app-model.readthedocs.io/en/latest/application/).
-`app-model` is a Python package that provides a declarative schema for an
-application.  It is an abstraction developed by napari developers, with the
+[App-model](https://app-model--142.org.readthedocs.build/en/142/) is a Python package
```

Comment:
No reason, I think I've had the app model docs open in a tab for a long time and theres been a new release since and when I copied the URL it was to an older version? No idea, I will fix!